go-1.19.yaml

```diff
@@ -11,6 +11,8 @@ package:
       attestation: TODO
       license: BSD-3-Clause
   dependencies:
+    provides:
+      - go=1.19
```

Comment:
I'm going to plead ignorance here about how `provides` works.

Why `.999`? Should this be `go=1.19.4`, and update to provide each new patch release? Should it do both, so that it counts as `go=1.19.<highest>` and `go=1.19.4` exactly?

Reply:
I would just do 1.19.999 tbh.  The version is just used to determine preference.  We need to do it specially for 1.19 because we have old packages.  In future we can just do `go=1.20`.

Reply:
Makes sense. Done.